lxd/container_lxc.go

```diff
@@ -1811,6 +1811,13 @@ func (c *containerLXC) initLXC(config bool) error {
 		return err
 	}
 
+	if lxc.HasApiExtension("seccomp_notify") && c.DaemonState().OS.SeccompListener {
```

Comment:
Should also be conditional so it won't run inside a truly privileged container (IsPrivileged && !inUserns)

Reply:
you mean (!IsPrivileged || inUserns)?